code/modules/response_team/ert.dm

```diff
@@ -177,6 +177,7 @@ GLOBAL_VAR_INIT(ert_request_answered, FALSE)
 	M.mind.original = M
 	M.mind.assigned_role = SPECIAL_ROLE_ERT
 	M.mind.special_role = SPECIAL_ROLE_ERT
+	M.mind.offstation_role = TRUE
```

Comment:
I don't think this is correct.
Offstation roles are things like dionae in terrarium, or syndicate agents in the syndicate lava base, who are *physically* unreachable (barring extraordinary effort).
ERT, however, are very much reachable, and are *on*station most of the time.
Setting this to TRUE just so that you can treat them the same in cult objective selection feels extremely hacky. Please don't. Better just check that special_role != SPECIAL_ROLE_ERT in the cult objective selector.

(And from the design perspective: I don't even think ERT members being sacrifice targets is a bad thing. Yes, they *can* get to z2, but they usually don't until the cult is dealt with. And if they do go to z2, they can take captain, NTR, or other "legit" sacrifice objective with them, so it doesn't even address the problem fully. But in either case, I'm more concerned about this from the code standpoint.)

Reply:
>I don't think this is correct.
Offstation roles are things like dionae in terrarium, or syndicate agents in the syndicate lava base, who are physically unreachable (barring extraordinary effort).
ERT, however, are very much reachable, and are onstation most of the time.
Setting this to TRUE just so that you can treat them the same in cult objective selection feels extremely hacky. Please don't. Better just check that special_role != SPECIAL_ROLE_ERT in the cult objective selector.

My main reason for not doing as you suggested was the description for the variable in `mind.dm`:
```cs
var/offstation_role = FALSE //set to true for ERT, deathsquad, abductors, etc, that can go from and to z2 at will and shouldn't be antag targets
```
It seems to me like it's intended for exactly this use, but someone just forgot to add ERT to the list.
Also, I feel like having a check specifically for the ERT role is a bit too snowflakey; Considering the above variable exists and does the same thing.
<hr>

>(And from the design perspective: I don't even think ERT members being sacrifice targets is a bad thing. Yes, they can get to z2, but they usually don't until the cult is dealt with. And if they do go to z2, they can take captain, NTR, or other "legit" sacrifice objective with them, so it doesn't even address the problem fully. But in either case, I'm more concerned about this from the code standpoint.)

I agree that it'd be good to have ERT members as targets for antags, but I've heard of two instances in the last month alone of the ERT leaving mid-round and the cult being stuck until an admin notices or someone ahelps.
While it's a lot more likely that they'll stay until the shuttle is called, it's definitely not certain.

Reply:
Setting it because the comment says so is fair.
I'm retracting my concern.